src/styles/utils/sizes.scss

```
@@ -0,0 +1 @@
+$layout-size: 942px;
```

нельзя задавать фиксированную ширину )
её надо вычислять. примерно так
```suggestion
$layout-size: 
  карточек в ряду * ширину карточки + гэп между карточками * (карточек в ряду - 1);
```